### ISSUE TITLE
better management on the left side (for faster access)

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [HelloZeroNet/ZeroHello#6]
I'd like more than reordering (1).

Currently on the left site in ZeroHello, I have my sites under "favorited sites", and all others are in "connected sites" (and this takes me time to find single ZeroNet sites).

What I'd need is: "my sites" where I put my sites in, and a collapsing/expanding for this "my sites" 
(collapsing, so I could reach sites faster, without moving the mouse much).

Also, I'd like many more such collapsable entries (where I can add sites), which I can name myself.

So, the structure could be like this:

"my sites"
"favorited sites"
"todo (read in future)"
"friends"
"connected sites"
"<add new entry>"

(1) https://github.com/HelloZeroNet/ZeroHello/issues/1

**Comment by HelloZeroNet** [comment]
Could be possible, but why not use the browser's favorite function?

**Comment by Erkan-Yilmaz** [comment]
I assume you mean the bookmark feature (1)
The restructure I'd like because I have the newsfeed in ZeroHello to stay always uptodate.

(1) I don't know I am not just using it, probably there might be a shortcut combination, so I don't need to go over the browser menu.

**Comment by kseistrup** [comment]
I agree with @Erkan-Yilmaz, I would also like to see user-customizable and collapsible category labels in the sidebar of ZeroHello – it would make the sidebar much more manageable and useful.

**Comment by slrslr** [comment]
yes, categorizing sites and custom sort enable/disable custom ordering in these categories + allow adding (or editting default site description).
+ Search/filtering input field will search not only names, but also custom descriptions and site addresses + contain a button to clear the field. (not everyone knows he can double click and start typing to overwrite previously typed content)
